apps/arbeidsforhold-api/src/main/java/no/nav/registre/testnorge/arbeidsforhold/repository/model/OpplysningspliktigModel.java

```diff
@@ -0,0 +1,54 @@
+package no.nav.registre.testnorge.arbeidsforhold.repository.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import java.util.Date;
+
+@Entity
+@Table(name = "OPPLYSNINGSPLIKTIG")
+@Builder
+@Getter
+@EntityListeners(AuditingEntityListener.class)
+@AllArgsConstructor
+@NoArgsConstructor
+public class OpplysningspliktigModel {
```

Comment:
Når det heter OpplysningspliktigModel tenkte jeg at det var organisasjonen som er opplysningspliktig. Er ikke dette mer en model for oppsummeringsdokumentet?

Reply:
Hmm ja det er sant